zbus/src/object_server/interface.rs

```diff
@@ -130,6 +132,15 @@ pub trait Interface: Any + Send + Sync {
     fn introspect_to_writer(&self, writer: &mut dyn Write, level: usize);
 }
 
+/// A newtype for a reference counted Interface trait-object, with a custom Debug impl.
```

Comment:
custom -> manual. Otherwise, it gives the impression that we need some special impl.